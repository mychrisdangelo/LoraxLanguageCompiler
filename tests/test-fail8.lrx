/* 
 * Author: Zhaarn Maheswaran
 * Tests incorrect tree decl
 */

int main () {
	
	tree <int> t (3);
	t = 1[2, 5[3, 5, 4, 3], 5];
	
}