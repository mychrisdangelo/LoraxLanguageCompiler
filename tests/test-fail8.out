Fatal error: exception Failure("Tree type is not consistent: expected <int>(3) but received <int>(4)")
